docs/src/docs/deploy.md

```diff
@@ -88,14 +88,14 @@ Kubernetes example:
 
 Going a step further you can use the Kit CLI container to create your own bespoke ModelKit containers.
 
-Example `dockerfile` for a custom container that has <my-modelkit> built into it:
+Example `dockerfile` for a custom container that has `my-modelkit` built into it:
```

Comment:
if you gonna use `<` and `>` it needs to be wrapped in a inline-code backtick, otherwise, it tries to render it as html and fails.